docs/guide/applications.md

```diff
@@ -0,0 +1,138 @@
+# Applications
```

Comment:
When I arrive on this screen, I get scrolled to the bottom of the screen

Reply:
Caution: the preview link isn't up-to-date.

I will take a closer look if I'm able to reproduce.

Reply:
Like @FredericEspiau, I have same behavior on each page, once you've scrolled down one

Reply:
@FredericEspiau @JeremyDec Found the issue.

I will update the preview link.